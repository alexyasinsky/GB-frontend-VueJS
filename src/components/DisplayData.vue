<template>
  <div>
    <div v-for="(item, index) in items" :key="index">
      {{ item }}
  </div>
</div>
</template>

<script>
  export default {
  name: "DisplayData",
  props: {
    items: {
      type: Array,
      default: ()=>[]
    }
  }
}
</script>