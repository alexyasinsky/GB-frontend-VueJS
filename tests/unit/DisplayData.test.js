import {mount} from '@vue/test-utils';
import DisplayData from "@/components/DisplayData";


describe('DisplayData Component',()=>{
  test('Test snapshot',()=>{
    const wrapper = mount(DisplayData);
    expect(wrapper).toMatchSnapshot();
  });
})