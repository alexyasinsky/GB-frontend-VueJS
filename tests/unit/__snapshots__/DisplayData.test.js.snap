// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisplayData Component Test snapshot 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
